docs/adding_packages/build_and_package.md

```diff
@@ -6,72 +6,13 @@ Both methods often use build helpers to build binaries and install them into the
-  Tools explicitly marked as available by users through conf like `tools.gnu:make_program`, `tools.gnu:pkg_config`, `tools.microsoft.bash:path`, `tools.microsoft.bash:subsystem` should be taken into account to conditionally inject a build requirement (these conf should have precedence over build requirement equivalent hardcoded in the recipe).
-
-* It is forbidden to run other conan client commands during build. In other words, if upstream build files call conan under the hood (through `cmake-conan` for example or any other logic), this logic must be removed.
-
-* Settings from profile should be honored (`build_type`, `compiler.libcxx`, `compiler.cppstd`, `compiler.runtime` etc).
-
-* Compiler paths from host profile should be honored and properly propagated to underlying build system during the build:
-
-  | compiler type | conf / env var |
-  |---------------|----------------|
-  | C compiler | `c` key of `tools.build:compiler_executables`, otherwise `CC` environment variable |
-  | C++ compiler | `cpp` key of `tools.build:compiler_executables`, otherwise `CXX` environment variable |
-  | ASM compiler | `asm` key of `tools.build:compiler_executables`, otherwise `CCAS` environment variable |
-  | CUDA compiler | `cuda` key of `tools.build:compiler_executables` |
-  | Fortran compiler | `fortran` key of `tools.build:compiler_executables`, otherwise `FC` environment variable |
-  | Objective-C compiler | `objc` key of `tools.build:compiler_executables` |
-  | Objective-C++ compiler | `objcpp` key of `tools.build:compiler_executables` |
-  | Resource files compiler | `rc` key of `tools.build:compiler_executables`, otherwise `RC` environment variable |
-  | Archiver | `AR` environment variable |
-  | Linker | `LD` environment variable |
-
-  They should be curated on the fly if underlying build system expects a specific format (no spaces in path, forward slash instead of back slash, etc).
-
-* These compiler and linker conf from host profile should be honored and propagated to underlying build system during the build:
-  * `tools.build:cflags`
-  * `tools.build:cxxflags`
-  * `tools.build:defines`
-  * `tools.build:sharedlinklags`
-  * `tools.build:exelinkflags`
-  * `tools.apple:enable_bitcode` (only if host OS is `iOS`/`watchOS`/`tvOS`)
-
-* Multithread build (if supported by underlying build system):
-  * if some steps are sensitive to race conditions, monothread should be enforced.
-  * otherwise multithreaded build should be enabled with a number of cores controlled by `tools.build:jobs` conf from host profile if it is set, otherwise to all cores of build machine.
-
-## Package Method
```

Comment:
Package method is well covered by Conan docs already:

- https://docs.conan.io/2/reference/conanfile/methods/package.html
- https://docs.conan.io/2/tutorial/creating_packages/package_method.html